src/Utils.js

```diff
@@ -7,16 +7,31 @@ import {
 } from './react-compat';
 import {
   REACT013,
-  REACT014,
+  REACT15,
 } from './version';
 
+function internalInstanceKey(node) {
+  return Object.keys(Object(node)).filter(key => key.match(/^__reactInternalInstance\$/))[0];
```

Comment:
@spicyj also every goPD and ownKeys in every realm that i could possibly create!